x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -24,48 +25,13 @@ import {
   ResolverProcessType,
   RelatedEventEntryWithStatsOrWaiting,
 } from '../types';
-import { SymbolIds, NamedColors } from './defs';
+import { SymbolIds, useResolverTheme, NodeStyleMap } from './defs';
 import { ResolverEvent } from '../../../common/endpoint/types';
 import { useResolverDispatch } from './use_resolver_dispatch';
 import * as eventModel from '../../../common/endpoint/models/event';
 import * as processModel from '../models/process_event';
 import * as selectors from '../store/selectors';
 
-const nodeAssets = {
```

Comment:
:grey_question:
Benefit: Keeps the node file cleaner.
Drawback: Indirection when you want/need to adjust node style or content.

I think this makes sense as it is here, just want to take score on that account.